pkg/instancetype/instancetype.go

```diff
@@ -855,21 +857,19 @@ func (m *InstancetypeMethods) inferDefaultsFromDataVolume(vm *virtv1.VirtualMach
 	}
 	dv, err := m.Clientset.CdiClient().CdiV1beta1().DataVolumes(vm.Namespace).Get(context.Background(), dvName, metav1.GetOptions{})
 	if err != nil {
-		// Handle garbage collected DataVolumes by attempting to lookup the PVC using the name of the DataVolume in the VM namespace
```

Comment:
Why do we need to drop the comments? There are several more similar changes further on.

Reply:
@vasiliy-ul I by mistake drop this comments